### ISSUE TITLE
error minio@7.0.27: The engine "node" is incompatible with this module. Expected version ">8  <16.8.0". Got "16.14.2"

### ISSUE THREAD

**Issue by r4dul** [minio/minio-js#1025]
npm view minio version
7.0.27
node -v               
v16.14.2

I get the error from the title when trying to create a docker image.
I'm using node:lts-alpine as base image, which uses node version: v16.14.2.

**Comment by r4dul** [comment]
I think I will answer myself: 
I see that in minio 7.0.26 in package.json we have:  
 "engines": {
    "node": ">= 4"
  }
In the case above minio package shall work with node version v16.14.2
While in version 7.0.27 we have in package.json: 
  "engines": {
    "node": ">8  <16.8.0"
  }
and that's why an error will be thrown.

While I saw that the issue was fixed here: https://github.com/minio/minio-js/pull/1018/commits/a5526caf86d12409e93a0ed7c6ec2da5d1511a8a (but still needs a new minio release I think)
my next question is:
Is it safe to use minio 7.0.26 with node version v16.14.2 or shall I wait until minio version 7.0.28 is released?
Thanks in advance!
